src/main/java/org/itsallcode/openfasttrace/importer/legacytag/LegacyTagImporterFactory.java

```diff
@@ -39,43 +40,61 @@
 
     private static LegacyTagImporterConfig defaultConfig = new LegacyTagImporterConfig();
 
-    private final List<PathConfig> pathConfigs;
+    private final Supplier<LegacyTagImporterConfig> config;
 
     public LegacyTagImporterFactory()
     {
-        this(defaultConfig.getPathConfigs());
+        this(() -> defaultConfig);
     }
 
-    public LegacyTagImporterFactory(final List<PathConfig> pathConfigs)
+    public LegacyTagImporterFactory(final LegacyTagImporterConfig config)
```

Comment:
Why support two types of constructors that are so similar in effect but use different paradigms?

Reply:
✔️